contracts/RAZOR.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Didn't we decide to use token contract from the original repo?

Reply:
I have used same contract but hardcoded intial supply to 1 billion